src/components/PropertyCard.tsx

```diff
@@ -43,8 +43,7 @@ const PropertyCardContainer = styled.div`
 const PropertyImage = styled.img`
     // prevent image from overflowing
     max-width: 100%;
-    max-height: 250px;
-    height: auto;
+    height: 250px;
```

Comment:
Ideally we'd probably want to avoid hard-coding these. But whatever. This is perfect for our needs.